clock:
    publisher_topic: clock
    publisher_Hz: 10

steering_servo:
    subscriber_topic: cmd_vel
    servo_ms:
        neutral_ms: 1500
        min_ms: 1200 # links
        max_ms: 1750 # rechts
        use_way: 100 # limit max and min steering angle, in percent [0, 100]

motor:
    subscriber_topic: cmd_vel

xbox:
    publisher_topic: cmd_vel
    publisher_Hz: 10
